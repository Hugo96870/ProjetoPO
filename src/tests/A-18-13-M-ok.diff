*** tests/A-18-13-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-18-13-M-ok.outhyp	2020-12-03 16:57:15.133066000 +0000
***************
*** 59,65 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|ELITE|3000|2820
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 59,65 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
***************
*** 85,91 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 2|R2|B|6|2400|2280|2|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 85,91 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: Visualizar: Operação inválida: A transacção '2' não existe.
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
