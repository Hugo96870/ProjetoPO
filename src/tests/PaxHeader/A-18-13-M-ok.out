30 mtime=1606841711.306162787
